src/key/key_map.rs

```diff
@@ -495,28 +499,22 @@ impl KeyMap {
             content
         };
 
-        let uncrypted_file = UncryptedVaultFile::new(file_content);
+        let unencrypted_file = UnencryptedVaultFile::new(file_content);
 
-        let crypted_file_content = Crypto::encrypt(pem.get_public_key(), &uncrypted_file)
-            .with_context(|| {
-                format!(
-                    "encryption of {} failed with key {}",
-                    &filepath,
-                    pem.get_name()
-                )
-            })?;
+        let encrypted_file_content = Crypto::encrypt(pem.public_key(), &unencrypted_file)
+            .with_context(|| format!("could not encrypt {filepath} with key {}", pem.name()))?;
```

Comment:
Here you pull the `filepath` into the format string, but 7 lines below you didn't we should stay consistent here (I personally like to only pull the identifier into the format string if all arguments can be pulled into the format string, but I'm indifferent to what you choose here).

Reply:
Fixed! Also found and fixed a whole bunch more via `cargo clippy --fix`.